index.html

```diff
@@ -37,8 +38,200 @@ <h1>I'm Joseph Glad to meet you!</h1>
+            <div class="canopy">
+                <p>CANOPY</p>
+            </div>
+            
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>Backend Dev</p>
+            </div>
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>2015</p>
+            </div>
+        </div>
+        <div class="content">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        
+         <div class="link">
+            <a href="">Html</a>
+            <a href="">css</a>
+            <a href="">javascript</a>
+         </div>
 
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
+    <section id="secondPage">
+        <img class="snap" src="Images/Snapshoot Portfolio (1).png" alt="">
+        <h2 class="tonic">Tonic</h2>
+        <div class="container">
+            <div class="canopy">
+                <p>CANOPY</p>
+            </div>
+            
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>Backend Dev</p>
+            </div>
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>2015</p>
+            </div>
+        </div>
+        <div class="content">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        
+         <div class="link">
+            <a href="">Html</a>
+            <a href="">css</a>
+            <a href="">javascript</a>
+         </div>
+
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
+    <section id="secondPage">
+        <img class="snap" src="Images/Snapshoot Portfolio (3).png" alt="">
+        <h2 class="multi">Multi-Post
+            Stories</h2>
+        <div class="container">
+            <div class="canopy">
+                <p>CANOPY</p>
+            </div>
+            
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>Backend Dev</p>
+            </div>
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>2015</p>
+            </div>
+        </div>
+        <div class="content">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        
+         <div class="link">
+            <a href="">Html</a>
+            <a href="">css</a>
+            <a href="">javascript</a>
+         </div>
 
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
+    <section class="about">
```

Comment:
- [ ] Kindly consider encompassing everything from here to the end `whole about section` in one `section` tag so meet the project requirement.
- [ ] Also consider applying display: flex in this section to layout the elements as it is one of the project requirement.

Reply:
- Please, just like the previous reviewers said, use the `flex` display property to align content of this section. This will allow for more responsiveness, please fix this 🙏